api/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -102,6 +102,11 @@ type OpenStackDataPlaneServiceSpec struct {
 	// +kubebuilder:default=false
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
 	AddCertMounts bool `json:"addCertMounts" yaml:"addCertMounts"`
+
+	// DeployOnAllNodesets - should the service be deploy across all nodesets
+	// This will override default target of a service play, setting it to 'all'.
+	// +kubebuilder:validation:Optional
+	DeployOnAllNodesets *bool `json:"deployOnAllNodesets,omitempty" yaml:"deployOnAllNodesets,omitempty"`
```

Comment:
this needs to be `DeployOnAllNodeSets` (NodeSet with capital S) for consistency with how we use NodeSets everywhere else.

Reply:
Thanks. done.